From e71816f6c9dbbe071baf43ffb28e05581d5c3aff Mon Sep 17 00:00:00 2001
From: Andreas Blaesius <skate4life@gmx.de>
Date: Sun, 10 Dec 2017 11:47:37 +0100
Subject: [PATCH] Bump Security String to 2017-12-01

- google forgot to update the security string while pushing android-7.1.2_r36 tag

Change-Id: I80594e14e4e6a1304e0650e0eb7fc0de80e3225c
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 9ff3565..0f77b26 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -131,7 +131,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2017-10-05
+    PLATFORM_SECURITY_PATCH := 2017-12-01
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.7.4

